.readthedocs.yaml

```diff
@@ -0,0 +1,21 @@
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+version: 2
+
+build:
+  os: ubuntu-22.04
+  tools:
+    python: "3.9"
```

Comment:
For Furo, I think you want to install the `fonts-roboto` package:

https://github.com/InsightSoftwareConsortium/itk-wasm/blob/d3f8778187748d66a97df35cf030ad73527204f7/.readthedocs.yml#L12-L13